peernode/src/peer-node/cli/tests.rs

```diff
@@ -0,0 +1,16 @@
+#[cfg(test)]
+mod tests {
+    use crate::cli;
+
+    #[test]
+    fn test_clap_cli() {
+        // Simulate command line arguments
+        let args = vec!["market", "set", "localhost:50051"];
+
+        // Call your CLI function with the arguments
+        let result = cli().get_matches_from(args);
+
+        // Assert the expected output
+        //assert_eq!(result, Some(_));
```

Comment:
Is this intended? The test will not be complete without assertion right? If you are planning on fixing this later then add a TODO comment.

Reply:
Got it, will do